<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q7FileWindow</class>
 <widget class="QWidget" name="Q7FileWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../R/Res.qrc">
    <normaloff>:/images/icons/cgSpy.gif</normaloff>:/images/icons/cgSpy.gif</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>400</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Selection">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>400</height>
       </size>
      </property>
      <attribute name="title">
       <string>Selection</string>
      </attribute>
      <widget class="QComboBox" name="direntries">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>531</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>15</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Current directory</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>45</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected file</string>
       </property>
      </widget>
      <widget class="QComboBox" name="fileentries">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>301</width>
         <height>27</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTreeView" name="treeview">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>680</width>
         <height>240</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>680</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>680</width>
         <height>240</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="bAction">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../R/Res.qrc">
         <normaloff>:/images/icons/save.gif</normaloff>:/images/icons/save.gif</iconset>
       </property>
      </widget>
      <widget class="QCheckBox" name="cShowAll">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>40</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show all</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bBack">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../R/Res.qrc">
         <normaloff>:/images/icons/arrow.gif</normaloff>:/images/icons/arrow.gif</iconset>
       </property>
      </widget>
      <widget class="QCheckBox" name="cShowDirs">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>40</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show directories</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>350</height>
       </size>
      </property>
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <widget class="QCheckBox" name="__O_filtercgnsfiles">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show .cgns files</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="__O_filterhdffiles">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show .hdf* files</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear file history</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear directory history</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Check history entries</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="__O_loadnodedisplay">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>274</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Load node on display request</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="__O_donotloadlargearrays">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>274</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Do not load large data arrays</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>217</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Max data size to load</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="__O_maxloaddatasize">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>50</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="__O_followlinksatload">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>90</y>
         <width>274</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Follow links during file load</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="__O_stoploadbrokenlinks">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>274</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop loading on broken link</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="__O_donotfollowlinksatsave">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>274</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Do not follow links during file save</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>203</x>
         <y>10</y>
         <width>20</width>
         <height>251</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>331</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Directory change sets the selection root</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../R/Res.qrc"/>
 </resources>
 <connections/>
</ui>
